test/api_integration/apis/core/compression.ts

```diff
@@ -41,5 +41,14 @@ export default function ({ getService }: FtrProviderContext) {
           expect(response.header).not.to.have.property('content-encoding');
         });
     });
+
+    it(`supports brotli compression`, async () => {
+      await supertest
+        .get('/app/kibana')
```

Comment:
Nit: should we explicitly test it with an API? Just in case /app and /api are served differently (as we do today with bundles)?